tests/fixtures/policies/ray_sampler_utils.py

```diff
@@ -0,0 +1,49 @@
+class MockAlgo():
+    def __init__(self, env_spec, policy, max_path_length=16):
+        self.env_spec = env_spec
+        self.policy = policy
+        self.max_path_length = max_path_length
+
+
+class ScriptedPolicy():
```

Comment:
can you attempt to use the mock library directly, rather than creating more test fixtures? i think they are overused in our test suite.

you should be able to use mock.side_effect to achieve what you want here.

Reply:
I'm not particularly opposed to test fixtures, as long as they're simple.
Having said that, in this case the only method which needs a non-trivial mock is `get_action`, so it shouldn't be too hard to use the Mocking library directly.

Reply:
It's worth mentioning that a specific disadvantage of `MagicMock` (and `PickleableMagicMock`), is that we don't have a way of pickling the mocking information itself (e.g. method return values). In comparison, text fixtures are generally trivially pickleable.